sdk/desktopvirtualization/Azure.ResourceManager.DesktopVirtualization/api/Azure.ResourceManager.DesktopVirtualization.netstandard2.0.cs

```diff
@@ -782,39 +848,76 @@ public partial class ScalingHostPoolReference
         public Azure.Core.ResourceIdentifier HostPoolId { get { throw null; } set { } }
         public bool? IsScalingPlanEnabled { get { throw null; } set { } }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct ScalingHostPoolType : System.IEquatable<Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType>
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
+        public ScalingHostPoolType(string value) { throw null; }
+        public static Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType Pooled { get { throw null; } }
+        public bool Equals(Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType other) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override bool Equals(object obj) { throw null; }
+        [System.ComponentModel.EditorBrowsableAttribute(System.ComponentModel.EditorBrowsableState.Never)]
+        public override int GetHashCode() { throw null; }
+        public static bool operator ==(Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType left, Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType right) { throw null; }
+        public static implicit operator Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType (string value) { throw null; }
+        public static bool operator !=(Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType left, Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolType right) { throw null; }
+        public override string ToString() { throw null; }
+    }
     public partial class ScalingPlanPatch
     {
         public ScalingPlanPatch() { }
         public string Description { get { throw null; } set { } }
         public string ExclusionTag { get { throw null; } set { } }
         public string FriendlyName { get { throw null; } set { } }
         public System.Collections.Generic.IList<Azure.ResourceManager.DesktopVirtualization.Models.ScalingHostPoolReference> HostPoolReferences { get { throw null; } }
-        public Azure.ResourceManager.DesktopVirtualization.Models.HostPoolType? HostPoolType { get { throw null; } set { } }
```

Comment:
This is a breaking change that is not allowed, please use [custom code](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices/src/Customized/Models/ServiceAccountEncryptionProperties.cs) to add this property back and mark it as Obsolete and EditorBrowsableState.Never